src/pcap/pcap.go

```diff
@@ -17,40 +23,27 @@ const (
 	// that requires both parties to be updated this value MUST be incremented by one. The calling
 	// party has to ensure that the compatibility level of the called party is equal or larger and
 	// refuse operation if it isn't.
-	CompatibilityLevel int64 = 0
+	CompatibilityLevel int64 = 1
```

Comment:
Any specific reason why this was bumped to one?

Reply:
Yes, discussed with you a couple weeks ago. We set the initial value to 1 because of unit tests.